### PR TITLE
VTCCA-5524 add BST connectors and config. add connector integration tests.  add feature switches

### PR DESCRIPTION
**Pull request by MarkAKelly**

### CODE REVIEW

File: app/uk/gov/hmrc/voapropertylinking/config/GuiceModule.scala [hunk 1]
```diff
@@ -62,25 +63,48 @@ class GuiceModule(
...
+        "voa.myClientPropertyLinks"            -> "bst.resources.externalPropertyLink.myClientPropertyLinks.path",
+        "voa.myClients"                        -> "bst.resources.externalPropertyLink.myClients.path",
+        "voa.createPropertyLink"               -> "bst.resources.externalPropertyLink.createPropertyLink.path",
+        "voa.createPropertyLinkOnClientBehalf" -> "bst.resources.externalPropertyLink.createPropertyLinkOnClientBehalf.path",
+        "voa.revokeClientsPropertyLink"        -> "bst.resources.externalPropertyLink.revokeMyClientsPropertyLink.path",
+        "voa.representationRequestResponse"    -> "bst.resources.authorisationManagementApi.representationRequestResponse.path",
+        "voa.agentAppointmentChanges"          -> "bst.resources.organisationManagementApi.agentAppointmentChanges.path",
+        "voa.myAgentDetails"                   -> "bst.resources.organisationManagementApi.myAgentDetails.path"
+      ),
+      servicesConfig.baseUrl("voa-modernised-api")
```
[the remark this answers is not in the document]
Comment (PeterPerhac) [reply app/uk/gov/hmrc/voapropertylinking/config/GuiceModule.scala:86]: I see you use `servicesConfig.baseUrl("voa-bst")` in an integration test below

**Comment by PeterPerhac** [comment]
👍 Looks good to me. I found one copy-paste issue I think, but other than that all looks good

File: app/uk/gov/hmrc/voapropertylinking/config/GuiceModule.scala (hunk 1 above)
[the remark this answers is not in the document]
Comment (MarkAKelly) [reply app/uk/gov/hmrc/voapropertylinking/config/GuiceModule.scala:86]: Yep, well spotted.  Updated

**Comment by zachhall1234** [comment]
LGTM 👍

**Pull request merged**